mlflow/tracing/utils/__init__.py

```diff
@@ -243,6 +243,21 @@ def generate_request_id() -> str:
     return uuid.uuid4().hex
 
 
+def construct_full_inputs(func, *args, **kwargs) -> dict[str, Any]:
```

Comment:
💯

Reply:
Actually, since we make it a standalone method, should we write tests for this method?